123_state_tasks.yml

```diff
@@ -25,24 +25,36 @@ targets:
       - MN_data
       - MI_data
       - IL_data
+      - IN_data
+      - IA_data
 
   # --- WI --- #
   
   WI_data:
-    command: get_site_data(oldest_active_sites, I('WI'), parameter)
+    command: get_site_data(I('1_fetch/tmp/inventory_WI.tsv'), parameter)
```

Comment:
to riff on my comment above - when you add I() around the file name, you are telling the pipeline to treat the text "1_fetch/tmp/inventory_WI.tsv" as the dependency, _not_ the file contents underlying that path. This may seem fine, since the contents aren't really going to change - but is that true? 

What if you leave this pipeline along for 30 days and add "OH" to your remake file? Well, that will trigger a rebuild of the inventory for _all_ states. Maybe that will result in a change to the inventory in the `1_fetch/tmp/inventory_WI.tsv` file (probably not though in reality). But, if that file changes, it won't be recognized as a change due to the use of I(). Make sense?

Reply:
yep. Thanks

Reply:
makes sense.